Functions/Example/Tests/FIRFunctionsTests.m

```diff
@@ -33,10 +33,43 @@ - (void)tearDown {
 - (void)testURLWithName {
   FIRFunctions *functions = [[FIRFunctions alloc] initWithProjectID:@"my-project"
                                                              region:@"my-region"
+                                                       customDomain:nil
                                                                auth:nil
                                                           messaging:nil];
   NSString *url = [functions URLWithName:@"my-endpoint"];
   XCTAssertEqualObjects(@"https://my-region-my-project.cloudfunctions.net/my-endpoint", url);
 }
 
+- (void)testRegionWithEmulator {
+  FIRFunctions *functions = [[FIRFunctions alloc] initWithProjectID:@"my-project"
+                                                             region:@"my-region"
+                                                       customDomain:nil
+                                                               auth:nil
+                                                          messaging:nil];
```

Comment:
These can be pulled into the test setup method to avoid repeating them in each test. You'll need to add a variable to the test class to store the `functions` object.

Reply:
Good idea, thanks.